unit_tests/test_create_robyn_app.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+import os
+from unittest.mock import patch
+from robyn.cli import create_robyn_app, SCAFFOLD_DIR, CURRENT_WORKING_DIR
+
+
+@pytest.fixture
+def project_directory():
+    return CURRENT_WORKING_DIR / "test_dir"
```

Comment:
Hey @Mr-Sunglasses 👋 

The windows tests are failing due to this path joining. Try doing `Path.resolve` here